modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java

```diff
@@ -491,6 +495,48 @@ private boolean _isWildcardValue(String value) {
 		return false;
 	}
 
+	private void _mapGlobalJSScriptElementAttributesToString(
+		ClientExtension clientExtension) {
+
+		Map<String, Object> typeSettings = clientExtension.typeSettings;
+
+		Map<String, Object> scriptElementAttributes =
+			(Map<String, Object>)typeSettings.get("scriptElementAttributes");
+
+		if (scriptElementAttributes == null) {
+			return;
+		}
+
+		Set<Map.Entry<String, Object>> entrySet =
+			scriptElementAttributes.entrySet();
+
+		Stream<Map.Entry<String, Object>> stream = entrySet.stream();
+
+		typeSettings.put(
+			"scriptElementAttributes",
+			stream.map(
+				entry -> {
+					StringBuilder stringBuilder = new StringBuilder();
+
+					stringBuilder.append(entry.getKey());
+					stringBuilder.append("=");
+
+					Object value = entry.getValue();
+
+					if (value instanceof Boolean) {
+						stringBuilder.append(value);
+					}
+					else {
+						stringBuilder.append(StringUtil.quote(value));
+					}
```

Comment:
This can be simplified to a single `if` check:

```java
Object value = entry.getValue();

if (!value instanceof Boolean) {
	value = StringUtil.quote(value);
}

stringBuilder.append(value);
```